.github/workflows/ci.yml

```diff
@@ -99,3 +101,30 @@ jobs:
       - name: pytest
         run: python3 -m pytest -x -v
 
+      # Note in order for ctest -T Coverage to work the tests have to be executed first
+      # with ctest. Moving this before the test step breaks coverage
+      - name: coverage
+        if: contains(matrix.compiler, 'g++-10')
+        run: |
+          ctest -T test -T Coverage
```

Comment:
The "ctest -T test -T Coverage" invocation may be a separate step if artifacts are preserved between steps.